doxia-sink-api/src/main/java/org/apache/maven/doxia/sink/Sink.java

```diff
@@ -1724,6 +1724,17 @@ public interface Sink extends AutoCloseable {
      */
     void comment(String comment);
 
+    /**
+     * Add a comment. The default implementation will just call {@link #comment(String)}.
+     *
+     * @param comment The comment to write.
+     * @param isBlockComment If <code>true</code> this is a block comment, i.e. nothing should follow on the same line
+     * @since 2.1
+     */
+    default void comment(String comment, boolean isBlockComment) {
+        comment(comment);
+    }
```

Comment:
I would rather name it the way it is: `void blockComment`

Reply:
I am still thinking about a more suitable name, because this only affects the suffix, but not the prefix of the comment (i.e. in case of `true` the comment is not necessarily the only thing in the line). Any good idea here?

Reply:
Probably `endsWithLineBreak` is more accurate here.